back/engines/commercial/user_custom_fields/app/policies/user_custom_fields/representativeness/r_score_policy.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module UserCustomFields
+  module Representativeness
+    class RScorePolicy < ApplicationPolicy
+      def show?
+        active? && admin?
```

Comment:
Is this definitely only available to admins?

Reply:
At least for now :) It might make sense to add moderators as well, but then we need to define what should be shown by default when opening the dashboard and no project has been selected yet.